index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<!-- Link CSS -->
+	<link rel="stylesheet" href="./styles/style.css">
+	<!-- Font Link -->
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap"
+		rel="stylesheet">
+	<title>Hernán Zamora Portfolio</title>
+</head>
+
+<body>
+
+	<!-- Nav Bar -->
+	<header>
+		<div class="nav-bar">
+			<a href="#">Hernán Zamora</a>
+			<!-- Nav -->
+			<nav class="menu">
+				<div class="bar"></div>
+				<div class="bar"></div>
+				<div class="bar"></div>
+			</nav>
+		</div>
+	</header>
+
+	<!-- Hero/Main Section -->
+	<main>
+			<!-- Heading - Text -->
+			<section class="main-container">
+				<h1 class="main-title">Hey there. I'm Hernán</h1>
+				<h2 class="main-subtitle">I'am a Software Developer</h2>
+				<p class="main-text">I can help you build a product , feature or website Look through some of my work and
+					experience!
+					If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+			</section>
```

Comment:
- [ ] **[OPTIONAL]** I think you missed this closing `</section>` here. Please consider removing it! Remember that the Headline section includes the Heading - Text plus the Social media images, so closing this section here wouldn't be the best. Also, you have another closing `</section>` at the end, so I assumed it was a typo. 🤓 🔧